docs/conf.py

```diff
@@ -124,6 +124,10 @@
     "python": ("https://docs.python.org/3", None),
 }
 
+# Settings for copybutton
+copybutton_prompt_is_regexp = True
+copybutton_prompt_text = r">>> |\.\.\. "  # doctest
```

Comment:
does this actually come up anywhere?  Why not include the full regex including the `[In]` etc to protect against that in the future

Reply:
Yeah good point, there are currently no doctests. But still, once there are (and I encourage adding them, because they can be tested with pytest as well), these lines make the copy button there more convenient.

Reply:
Wanted to add the [`--doctest-modules`](https://docs.pytest.org/en/6.2.x/doctest.html) flag to `pytest.ini`, but then noticed this 'bug' #210.

Reply:
doctest = testing snippets in docstrings and in `.md` and `.rst` files? The notebooks that make up the majority of the docs are already included in the test runs.

Reply:
Yes, the ones in md and rst files would be doctests too, but I initially only had doctests in the docstrings (that is, the API) in mind. It's in line with #203, where the idea is that the API can serve more as its own documentation as well (with some integrated examples). That said, I now start to wonder how useful doctests are for a matplotlib framework..
Let's move this to #208.